addons/account_edi_ubl_cii/models/account_edi_xml_ubl_bis3.py

```diff
@@ -257,6 +257,12 @@ def _export_invoice_constraints(self, invoice, vals):
             'peppol_endpoint_is_set_supplier':  self._check_required_fields(vals['supplier'], 'peppol_endpoint'),
             'peppol_endpoint_is_set_customer':  self._check_required_fields(vals['customer'], 'peppol_endpoint'),
         })
+        for line_vals in vals['vals']['invoice_line_vals']:
+            constraints.update({
+                'peppol_item_name_is_set': self._check_required_fields(line_vals['item_vals'], 'name', _("Every invoice line should have an item name.")),
+            })
+            if constraints['peppol_item_name_is_set']:
+                continue # once one line has already failed the check, we can stop checking further
```

Comment:
That part is useless^^
Actually, you need to add a constraint for each line so the name of your constraint should be `peppol_item_name_is_set_<line_id>` or something. Otherwise, you are overriding the same constraint every time